source/guides/getting-started/quick-start.html.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Guides | Quick Start
+next_guide:
+  title: Interactors Introduction
+  url: /guides/interactors/introduction.html
+---
+
+# Quick Start
+
+This guide will teach you how to add BigTest to your React application.
+
+We'll cover:
+- Installing BigTest
+- Generating support files
+- Set Up
+- Writing some basic tests
+
+The steps outlined in this guide assume that you are using:
+
+- [ Mocha ](https://mochajs.org/)
+- Either Webpack or Parcel as your bundler.
+
+## Install Dependencies
+
+In your terminal, type in the following commands:
+
+```bash
+ yarn add --dev @bigtest/cli
+```
+
+```bash
+ yarn add --dev @bigtest/interactor
+```
+
+```bash
+ yarn add --dev @bigtest/react
+```
+
+- [ @bigtest/cli ](https://github.com/bigtestjs/cli) will give you access to the `bigtest` commands that we'll be using throughout the guide.
+
+- [ @bigtest/interactor ](https://github.com/bigtestjs/interactor) allows your tests to interact with your app the way your users will. They will wait for elements to exist before interacting with them, which means that you don’t have to worry about timing your tests correctly to sync up with any run loops. (Who even has time for that?)
```

Comment:
I'd use "appear" here instead of "exist." Since that feels more like how the user would perceive it.